plugins/imfile/imfile.c

```diff
@@ -1855,6 +1883,7 @@ addInstance(void __attribute__((unused)) *pVal, uchar *pNewVal)
 		}
 	}
 	inst->trimLineOverBytes = cs.trimLineOverBytes;
+	inst->ignoreOlderThan = cs.ignoreOlderThan;
```

Comment:
please remove support for the old config system - we do NOT provide new features via the old system.